modules/fundamental/src/sbom/endpoints/test.rs

```diff
@@ -30,29 +30,86 @@ async fn fetch_unique_licenses(ctx: &TrustifyContext) -> Result<(), anyhow::Erro
     let req = TestRequest::get().uri(&uri).to_request();
     let response: Value = app.call_and_read_body_json(req).await;
     let expected_result = &vec![
+        "(LicenseRef-0 OR LicenseRef-AFL) AND LicenseRef-0".to_string(),
+        "(LicenseRef-10 OR LicenseRef-0) AND LicenseRef-6".to_string(),
+        "(LicenseRef-4 OR LicenseRef-Artistic) AND LicenseRef-5 AND LicenseRef-UCD".to_string(),
```

Comment:
I wasn't expecting the `LicenseRef` to show up and I'm sure we don't want this as evaluated meaningless to the user but it should rather show the license name (ref. `If the license stored against the package is an internal licenseId then it will contain the corresponding name` in [TC-1902](https://issues.redhat.com/browse/TC-1902?focusedId=27095913&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27095913)).
Furthermore, checking the `LicenseRef-Artistic` for example, I can not find anyway its name, i.e. `Artistic`, in the previous test so it looks like, in a way or another, there's a bug in the code in the current `main` branch.

Reply:
First, it is very difficult to determine whether a license expression is a license reference before parsing it.
[1]  https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/
Second, how should we handle the cases("MIT AND LicenseRef-2 AND LicenseRef-CC-BY AND LicenseRef-GPLv3") where a license expression includes both an SPDX license ID (such as MIT) and a license reference?